asciidoctor-stylesheet/sass/devonfw.scss

```diff
@@ -71,6 +71,19 @@ hr {
   }
 }
 
+
+.sect3 {
+  margin-bottom: 100px
```

Comment:
I find 100px a bit too much. I would vote for 50px. And why there is no space at the top. I would expect this.
So maybe change this to `margin-top: 50px;` and remove `.sect3:last-of-type`

Reply:
Yeah, 50px seems totally fine.
When I go down to 50px I would also expect such space between the headline and the chapter title. 
As you mentioned, I can remove .sect3:last-of-type after changing the margin from bottom to top.

Reply:
As agreed on at our Review today, I just reduced the margin to 50px.